thirdparty/tinkerpop/src/test/java/org/calrissian/accumulorecipes/thirdparty/tinkerpop/BlueprintsGraphStoreTest.java

```diff
@@ -113,7 +120,7 @@ public void testVertices() {
     }
 
     @Test
-    public void testEdges() {
+    public void testEdges() throws TableNotFoundException {
```

Comment:
You know intelli-j allows you to add checked exceptions to the method signatures by default. This is what's done everwhere else... why is this different?